dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/post-fee-record-correction.controller/helpers.ts

```diff
@@ -0,0 +1,82 @@
+import { getFormattedReportPeriodWithLongMonth, mapReasonToDisplayValue, RecordCorrectionReason, ReportPeriod } from '@ukef/dtfs2-common';
+import externalApi from '../../../../../external-api/api';
+import EMAIL_TEMPLATE_IDS from '../../../../../constants/email-template-ids';
+import { FeeRecordCorrectionRequestEmails } from '../../../../../types/utilisation-reports';
+import { getBankById } from '../../../../../repositories/banks-repo';
+import { NotFoundError } from '../../../../../errors';
+
+export const formatReasonsAsBulletedListForEmail = (reasons: RecordCorrectionReason[]) => {
+  return reasons.map((reason) => `*${mapReasonToDisplayValue(reason)}`).join('\n');
```

Comment:
do we need a space between * and ${mapReasonToDisplayValue(reason)}?

Reply:
and could this just be ` => reasons.map((reason) => `*${mapReasonToDisplayValue(reason)}`).join('\n');`

Reply:
We don't need a space no:
![image](https://github.com/user-attachments/assets/0a938ae9-b773-4cbb-8078-5f5cc95d5642)